api/src/main/java/io/strimzi/api/kafka/model/KafkaRebalanceSpec.java

```diff
@@ -55,6 +64,47 @@ public void setSkipHardGoalCheck(boolean skipHardGoalCheck) {
         this.skipHardGoalCheck = skipHardGoalCheck;
     }
 
+    @Description("The upper bound of ongoing partition replica movements going into/out of each broker")
+    @Minimum(0)
+    public int getConcurrentPartitionMovementsPerBroker() {
+        return concurrentPartitionMovementsPerBroker;
+    }
+
+    public void setConcurrentPartitionMovementsPerBroker(int movements) {
+        this.concurrentPartitionMovementsPerBroker = movements;
+    }
+
+    @Description("The upper bound of ongoing partition replica movements between disks within each broker")
+    @Minimum(0)
+    public int getConcurrentIntraBrokerPartitionMovements() {
+        return concurrentIntraBrokerPartitionMovements;
+    }
+
+    public void setConcurrentIntraBrokerPartitionMovements(int movements) {
+        this.concurrentIntraBrokerPartitionMovements = movements;
+    }
+
+    @Description("The upper bound of ongoing partition leadership movements")
+    @Minimum(0)
+    public int getConcurrentLeaderMovements() {
+        return concurrentLeaderMovements;
+    }
+
+    public void setConcurrentLeaderMovements(int movements) {
+        this.concurrentLeaderMovements = movements;
+    }
+
+    @Description("The upper bound, in bytes per second, on the bandwidth used to move replicas")
+    @Minimum(0)
+    public long getReplicationThrottle() {
+        return replicationThrottle;
+    }
+
+    public void setReplicationThrottle(long bandwidth) {
+        this.replicationThrottle = bandwidth;
+    }
```

Comment:
do all these parameters have defaults? Are we forcing the user to set them even at 0? How was it working until today without this configuration?

Reply:
They all have defaults in CC, which are in the documentation. If the keys are not present or set to zero they are ignored (which I should add to the documentation).

Reply:
but setting `Minimum(0)` are you sure that the API server validates as ok a `KafkaRebalance` without these settings? I don't remember tbh.

Reply:
I have tested it with and without configuration and it works fine.